homeassistant/components/samsungtv/__init__.py

```diff
@@ -27,7 +27,7 @@ def ensure_unique_hosts(value):
                     {
                         vol.Required(CONF_HOST): cv.string,
                         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-                        vol.Optional(CONF_PORT): cv.port,
+                        vol.Optional(CONF_PORT): cv.deprecated,
```

Comment:
This should still be `cv.port`. We need to wrap this dict in the schema with `cv.deprecated` and pass the correct arguments to it. Search the code for some examples.